SharePointFramework/PortfolioWebParts/src/components/PortfolioOverview/PortfolioOverviewCommands/index.tsx

```diff
@@ -173,7 +173,17 @@ export class PortfolioOverviewCommands extends Component<
     this.setState({ isExporting: true })
     try {
       const { fltItems, fltColumns, selectedItems } = this.props
+
       const items = isArray(selectedItems) && selectedItems.length > 0 ? selectedItems : fltItems
+
+      fltColumns.map((col) => {
```

Comment:
@Rundez You're using `Array.map` without returning anything. Seems like it's `Array.forEach` you want to use here.